models/firewall.js

```diff
@@ -1,7 +1,10 @@
 import { TooManyRequestsError } from 'errors';
 import database from 'infra/database.js';
+import content from 'models/content.js';
 import event from 'models/event.js';
 
+import balance from './balance.js';
```

Comment:
Provavelmente essa importação não será necessária, mas seguindo o padrão...

```suggestion
import balance from 'models/balance.js';
```